### PR TITLE
Login

### PR DESCRIPTION
**Pull request by mohammedshanu**

### CODE REVIEW

**Review by R-Kiran-Raj — CHANGES_REQUESTED** [review]

File: tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    './src/**/*.{js,jsx,ts,tsx}', // Adjust this path according to your project structure
+  ],
+  theme: {
+    screens: {
```
[the remark this answers is not in the document]
Comment (mohammedshanu) [reply tailwind.config.js:7]: changed extra resolution

File: src/components/BlackIcons.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import linkedin from "../assets/blinkedin.png";
+
+function BlackIcons() {
+  return (
+    <div className="flex flex-row w-[65.44px] justify-between ">
+           <a  href='https://x.com/?lang=en'><img
+            src={twitter}
+           
+            alt="Example"
+            className="  w-[11.59px] h-[11.59px]  "
```
[the remark this answers is not in the document]
Comment (mohammedshanu) [reply src/components/BlackIcons.jsx:13]: removed the height and width

File: src/components/BlackIcons.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import facebook from "../assets/bfacebook.png";
+import twitter from "../assets/btwitter.png";
+import linkedin from "../assets/blinkedin.png";
+
+function BlackIcons() {
```
[the remark this answers is not in the document]
Comment (mohammedshanu) [reply src/components/BlackIcons.jsx:6]: component has renamed

File: src/components/whiteIcons.jsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import facebook from "../assets/facebook.png";
+import twitter from "../assets/twitter.png";
+import linkedin from "../assets/linkedin.png";
+
+function whiteIcons() {
```
[the remark this answers is not in the document]
Comment (mohammedshanu) [reply src/components/whiteIcons.jsx:6]: now changed to single socialMediaIcon component

**Pull request merged**